LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 LogDNA
+Copyright (c) 2017 LogDNA
```

Comment:
Why are we using 2017 here?

Reply:
Nvm, resolving this was answered in a previous conversation `Set copyright year to 2017 as it should be set to the year of original publication. That was the year of the first release on PyPi`